internal/commands/cmd_clone.go

```diff
@@ -1,14 +1,81 @@
 package commands
 
 import (
+	"context"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/google/go-github/github"
 	"github.com/spf13/cobra"
+	"libs.altipla.consulting/errors"
 )
 
+var org string
+var pattern string
+
 var cmdClone = &cobra.Command{
 	Use:     "clone",
 	Short:   "Clona uno o varios proyectos en local",
 	Example: "sites clone",
 	RunE: func(cmd *cobra.Command, args []string) error {
+		var repositoryNames []string
+		var dir []string
+
+		//***LEER PROYECTOS DE LA ORG***
+		client := github.NewClient(nil)
+
+		opt := &github.RepositoryListByOrgOptions{Type: "public"}
+		repos, _, err := client.Repositories.ListByOrg(context.Background(), org, opt)
```

Comment:
Inicializa una variable ctx arriba del todo. Los contextos no se van inicializando todo el rato sino que se usa el mismo.

```suggestion
		repos, _, err := client.Repositories.ListByOrg(ctx, org, opt)
```